public/scripts/app.js

```diff
@@ -1,24 +1,18 @@
 console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
-var $p_id;
+  var $profile_id;
 
-$("#addProject").fadeOut();
-$("#addVacation").fadeOut();
+  $("#addProject").fadeOut();
+  $("#addVacation").fadeOut();
 
-$('ul.nav li').click(function(){
-
-    console.log("event fired");
-    console.log($(this));
+  $('ul.nav li').click(function(){
     $(this).addClass('active');
     $(this).siblings().removeClass('active');
 
-   if(($(this).context.textContent)=="About")
-   {
-     $("#addProject").fadeOut();
-     console.log("About section");
-     //about section
-     $.ajax({
+    if(($(this).context.textContent)=="About") {
```

Comment:
Open your curly braces on the same line as your `if` statement, and indent all the code after it within these curly braces (similar to how you would indent code between a function's curly braces).